services/GraphQL/mutations/users.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from '@apollo/client'
+
+// create user
+export const CREATE_USER = gql`
```

Comment:
nit: Maybe would be good to change the name from CREATE to INSERT?

Reply:
I used INSERT, as you suggested